Plugins/GitUIPluginInterfaces/GitUIEventArgs.cs

```diff
@@ -10,12 +11,17 @@ public GitUIEventArgs(IWin32Window? ownerForm, IGitUICommands gitUICommands)
         {
             OwnerForm = ownerForm;
             GitUICommands = gitUICommands;
+            _refs = new(GitModule);
         }
 
         public IGitUICommands GitUICommands { get; }
 
         public IWin32Window? OwnerForm { get; }
 
         public IGitModule GitModule => GitUICommands.GitModule;
+
+        private readonly FilteredGitRefsProvider _refs;
```

Comment:
The interface could be used here - though pretty academic because the ctor must create an instance of the implementation class.
```suggestion
        private readonly IFilteredGitRefsProvider _refs;
```